docs/en/setup/README.md

```diff
@@ -0,0 +1,23 @@
+# Setup
+First and most important thing is, SkyWalking Satellite startup behaviours are driven by configs/satellite_config.yaml. Understood the setting file will help you to read this document.
+
+## Startup script
+The startup script is /bin/skywalking-satellite-{version}-{plateform}-amd64. 
+1. Generate docs.
+```shell script
+./bin/skywalking-satellite-{version}-{plateform}-amd64 docs --output=./docs
+```
```

Comment:
Why need generate doc in the setup part?

Reply:
just descripe the supported command

Reply:
maybe should move it to the other part